api/incidents/views.py

```diff
@@ -54,3 +56,26 @@ def create_redflag_record(self, request):
     def create(self, request):
         request.data['createdBy'] = request.user.id
         return self.validate_record(request)
+
+    def destroy(self, request, **kwargs):
```

Comment:
The name of this method is not very descriptive. You can give it a better name like `delete`

Reply:
I am using the destroy word to override the pre-existing method. I would love to request that it remains as to destroy for this approach to work.

Reply:
Ok